src/main/java/Task.java

```diff
@@ -0,0 +1,18 @@
+public class Task {
+
+    protected String description;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    public String toString(boolean isDone) {
+        String doneSymbol;
+        if (isDone) {
+            doneSymbol = "X";
```

Comment:
Suggest to change "doneSymbol" to be the class attribute, since you may want to persist the state of the variable after an instance of the class is created.